temporalcli/commandsmd/commands.md

```diff
@@ -4,173 +4,390 @@ Commands for the Temporal CLI.
+  ```
+    
+  For example:
+  
+  ```
+  Includes options set for [client](#options-set-for-client).
+  ```
+    
+  An options set declaration is the equivalent of pasting those options into the bulleted options list.
+
+  Note: Command overlap or similarity does not require option sets.
+  Reserve option sets for parallel functionality.
+  Copy/paste is fine, otherwise.
+
+OPTION LISTING
+
+* List one option per line, using asterisk-delimited bullets.
+* Order the options with most commonly used options first.
+* Use this format:
+
+  ```
+  `--<long-option>`( , `-<short option>`) <data-type> - <short-description>( <extra-attributes>)
+  ```
+
+* Each option listing includes a long option with a double dash and a meaningful name.
+* [Optional] A short option uses a single dash and a short string.
+  When used, separate the long and short option with a comma and a space.
+* Backtick every option and short description. 
+  Include the dash or dashes within the ticks.
+  For example: `` `--workflow-id`, `-w` ``
+* A data type follows option names indicating the required value type for the option.
+  The type is `bool`, `duration`, `int`, `string`, `string[]`, `string-enum`, or `timestamp`. (_TODO: more_.)
+  Always parenthesize data types.
+  For example: `` `--raw` (bool) ``
+* A dash follows the data type, with a space on either side. 
+* The short description is free-form text and follows the dash.
+  Take care not to match trailing attributes. 
+  Newline wrapping and/or two-space indention condenses to a single space.
+* [Optional] extra attributes include:
+  * `Required.` - Marks the option as required.
+  * `Default: <default-value>.` - Sets the default value of the option. No default means zero value of the type.
+  * `Options: <option>, <option>.` - Sets the possible options for a string enum type.
+  * `Env: <env-var>.` - Binds the environment variable to this flag. For example: `Env: TEMPORAL_ADDRESS`.
 
 -->
 
-### temporal: Temporal command-line interface and development server.
+### temporal: Temporal command-line interface and development server
+
+The Temporal CLI (Command Line Interface) provide a powerful tool to manage, 
+monitor, and debug your Temporal applications. It also lets you run a local
+Temporal Service directly from your terminal. With this CLI, you can 
+start Workflows, pass messages, cancel application steps, and more.
+
+* Start a local development service: 
+      `temporal server start-dev` 
+* Help messages: pass --help for any command:
+      `temporal activity complete --help`
+
+Read more: https://docs.temporal.io/cli
 
 <!--
 * has-init
 -->
 
 #### Options
 
-* `--env` (string) - Environment to read environment-specific flags from. Default: default. Env: TEMPORAL_ENV.
-* `--env-file` (string) - File to read all environments (defaults to `$HOME/.config/temporalio/temporal.yaml`).
+* `--env` (string) - Active environment name. Default: default. Env: TEMPORAL_ENV.
```

Comment:
Let's discuss this. I'm not going to resolve this until the merits are weighed, as I believe "Active environment name" (what's being set for key-value) is a better than "Use settings from the `ENV` environment" (imperative-verb rule, which is wordy and describes how the key-value pair is used)

Reply:
Sure, I am okay deferring to you on editorial things like this. That's why I want your help!

Though note that without mentioning `\`ENV\``, you'll get output that looks like:

```
--env string    ...
```

rather than:

```
--env ENV    ...
```

Reply:
Sounds good. Key/Value is inherently noun. Thanks for this.

Reply:
Played around with 

```
      --env string                                        Active environment name. (default "default")
      --env ENVIRONMENT_NAME                              Active environment name (ENVIRONMENT_NAME). (default "default")
      --env ENV                              Active environment name (ENV). (default "default")

```

I think the first ('string') is the most understandable as it doesn't make ENV sound like it means anything other than a metasyntactic use